packages/adapter-cloudflare/src/middleware.js

```diff
@@ -0,0 +1,31 @@
+import { normalizeUrl } from '@sveltejs/kit';
+import { onRequest as user_middleware } from 'MIDDLEWARE';
+
+/**
+ * @param {{ request: Request, next: (init?: any, options?: any) => any}} context
+ */
+export async function onRequest(context) {
+	const { url, wasNormalized, denormalize } = normalizeUrl(context.request.url);
```

Comment:
I think we didn't mention `normalizeUrl` at all in the middleware docs; should we?

Reply:
Actually wow, yeah, `normalizeUrl` isn't anywhere in the docs that I can find, we definitely need to mention it _somewhere_ right?

Reply:
We don't, because it's not something you need to be concerned with when writing a middleware - because we normalize it for you. But it will be within the SvelteKit reference docs with the docs being what the jsdoc is right now